src/main/resources/org/broadinstitute/dsde/rawls/liquibase/changesets/20160609_all_attributes_column.xml

```diff
@@ -0,0 +1,89 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
database changes: 
* add and populate all_attribute_values column on entity table which is a space separated representation of all the non-list attributes of an entity - used for filtering
* add and populate list length on attribute tables - used for sorting
* add unique_entity_attribute because we should have it but this requires setting a length on the name attribute and the length can't be too big either or it won't fit in the index max key length